annotations/helm-annotations/src/main/resources/NOTES.txt

```diff
@@ -0,0 +1,73 @@
+# Dekorate Autogenerated Notes
+
+To access the Helm annotations or properties you just need to have the following dependency in your
+class path:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>helm-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+Build the project using:
+
+    mvn clean install
+
+You can find the generated Helm artifacts under: `target/classes/META-INF/dekorate/helm/<chart name>/` that should look like:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+
+**Note**: The `<chart name>` is set from either the property `dekorate.helm.name` or the `@HelmChart` annotation.
+
+How can it be used?
+
+First, make sure you have installed [the Helm command line](https://helm.sh/docs/intro/install/) and have connected into a cluster.
+
+Then, run the following Maven command in order to generate the Helm artifacts and build/push the image into a container registry:
+
+```shell
+mvn clean package -Ddekorate.push=true -Ddekorate.docker.registry=<container registry url> -Ddekorate.docker.group=<your group>
+```
+
+This command will push the image to a container registry and will become available when a pod or container is created.
+
+Finally, let's use Helm to deploy it into the cluster:
+
+```shell
+helm install helm-example ./target/classes/META-INF/dekorate/helm/<chart name>
+```
+
+The above command will deploy a chart using the default values (as defined within the `values.yaml` file). We can override the default values to use your `values.dev.yaml` file by doing:
```

Comment:
by doing --> by executing the following command:

Reply:
Changed